### DISCUSSION TITLE
How to know which version of Birdnet-Pi is running on my PI ?

### DISCUSSION THREAD

**Discussion by croisez** [mcguirepr89/BirdNET-Pi#546] [General]
There is no mention anywhere (at least I did'nt found it yet) of the version currently running.
How could I know current version?
In some screenshots of Birdnet-PI, I saw mention of a version near the title at upper area of HTML main file.
But now it is not displayed.
Any clue?
Thx
Louis.

**Comment by mcguirepr89** [comment]
Hi, @croisez -- I'm embarrassed to say that versioning is a silly joke for BirdNET-Pi -- basically, there are a million version (one for every time I pushed changes) and that it's not been tracked in any systematic way.

I apologize that this removes a lot of your agency as a user -- for what it may be worth, you are able to check how many upstream changes there are compared to your system by going to "Tools" > "System Controls" where you will see the "Update" button and (likely) a red indicator with number inside indicating how many commits your installation is behind `main`.

I recommend backing up your database `~/BirdNET-Pi/scripts/birds.db` before your update, since updates are another thing users can't really rely upon (I cross my fingers during updates and then fix what breaks -- SORRYYYYYYYY)

If you do see any "version" number on a web page, at this stage it will only mean that the installation is very old, since the versioning on the main page went away a long time ago (several months).

Sorry to only have a crap response :/

**Reply by ehpersonal38** [reply]
Will add to this with an equally crappy way of estimating the version. If you view the source of any page you should see this, where v= indicates the date in which style.css was last updated. Historically it's been updated on an at least monthly basis, so you can kinda work out a general version

![chrome_JVNXGnXx75](https://user-images.githubusercontent.com/103586016/190489774-8ae97f80-396b-4b59-bd1d-1ae195dd1a97.png)

**Comment by croisez** [comment]
Hi Patrick, hi @ehpersonal38,
first of all, Patrick, you don't have to apologize. I am very pleased to use BirdNET-Pi as is, and it brings me the joy to discover birds I even did not suspect being in my proximity.
It is your project, and you manage it as you want. I am sure it takes you much much time and energy. 
Be sure that every remarks or idea coming from myself will be gentle and positive.
And if I can help you for a specific task, just ask.
